programs/test/benchmark.c

```diff
@@ -732,6 +736,42 @@ int main(int argc, char *argv[])
         mbedtls_aes_free(&aes);
     }
 #endif
+#if defined(MBEDTLS_CIPHER_MODE_CFB)
+    if (todo.aes_cfb128) {
+        int keysize;
+        size_t iv_off = 0;
+        mbedtls_aes_context aes;
+        mbedtls_aes_init(&aes);
+        for (keysize = 128; keysize <= 256; keysize += 64) {
+            mbedtls_snprintf(title, sizeof(title), "AES-CFB128-%d", keysize);
+
+            memset(buf, 0, sizeof(buf));
+            memset(tmp, 0, sizeof(tmp));
+            CHECK_AND_CONTINUE(mbedtls_aes_setkey_enc(&aes, tmp, keysize));
+
+            TIME_AND_TSC(title,
+                         mbedtls_aes_crypt_cfb128(&aes, MBEDTLS_AES_ENCRYPT, BUFSIZE,
+                                                  &iv_off, tmp, buf, buf));
```

Comment:
CFB can processing random buffer sizes, which are not times of 16, should we consider about that in benchmark?

Reply:
The benchmark program uses a fixed size for block ciphers. It doesn't benchmark multiple sizes. Benchmarking multiple sizes could be a useful improvement for a benchmark program (encrypting 1000 10-byte messages and encrypting 1 10000-byte message have different performance profiles) but that would be a major task and not one that I'd consider high priority.